projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -224,20 +224,20 @@ export interface GridColumn<R> {
      * TODO: Should this be made to work with top level search on grids across all columns?
      *  The above to-do is going to be worked on as part of https://jira.eng.vmware.com/browse/VDUCC-27 and
      */
-    filterRendererSpec?: FilterRendererSpec<FilterConfig<unknown>>;
-
-    /**
-     * To enable only sorting without filtering on the column. This is because, passing in clrDgField turns both filtering
-     * and sorting feature on. And we want filtering to be off on some columns while still having sorting enabled.
-     */
-    sortBy?: string;
+    filter?: FilterRendererSpec<FilterConfig<unknown>>;
 
     /**
      * The configuration for the cliptext in the datagrid.
      * Defaults to size: 'lg', mouseoutDelay: undefined.
      * If null, will disable cliptext
      */
     cliptextConfig?: CliptextConfig;
+
+    /**
+     * Boolean to turn off sorting on the column. Adding a {@link GridColumn.queryFieldName} by default turns both
+     * filtering and sorting on and this switch allows for turning on just the filtering
+     */
+    notSortable?: boolean;
```

Comment:
Can this be called sortable instead? `!notSortable` looks weird. To default it consider adding a getter that tests if the property exists

```ts
/**
 *  Whether a columns is sortable, defaults to true
 */
isColSortable(col: SomeType) {
   return 'sortable' in col ? col.sortable : true; 
  // or
  return !(col.sortable === false)
}
```